miro_adapter/run_miro_adapter.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+set -o errexit
+set -o nounset
+
+python3 miro_adapter/build_json_derivatives.py \
```

Comment:
We don't always want to do this step do we? In fact could we split this into two even smaller services?

Reply:
Extreme microsizing!

Not sure it’s worth the overhead of another service, but we could check if it’s up-to-date before running this step (writes are atomic, so last modified time is last completed time).

Reply:
Ah, but what if this sort of JSON manifest is how we do all the adapters?

An adapter could watch the files, and do the DynamoDB step every time they change!

Reply:
Did this get addressed?

Reply:
No.